### PR TITLE
README change for vanilla deployment

### PR DESCRIPTION
**Pull request by akartsky**
README change for vanilla deployment

### CODE REVIEW

File: docs/deployment/vanilla/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Deploying Kubeflow on EKS
 
-This guide describes how to deploy Kubeflow on AWS EKS
+This guide describes how to deploy Kubeflow on AWS EKS. This vanilla version has very minimal changes to the upstream Kubeflow manifests. Here are the changes  
+- Kubeflow Notebook configuration has been modified to include some [custom images](../../../components/notebook-dockerfiles) built on [AWS DLC images](https://github.com/aws/deep-learning-containers/blob/master/available_images.md)
```
Comment (goswamig) [inline docs/deployment/vanilla/README.md:4]: `custom container image`

**Review by goswamig — APPROVED** [review]

**Comment by surajkota** [comment]
One minor request: this heading in Add-ons is strange: https://github.com/awslabs/kubeflow-manifests/tree/main/docs/deployment#vanilla-version-with-dex-for-auth-and-ebs-volumes-as-pv

can you maybe change it to just `Vanilla version`

File: docs/deployment/vanilla/README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Deploying Kubeflow on EKS
 
-This guide describes how to deploy Kubeflow on AWS EKS
+This guide describes how to deploy Kubeflow on AWS EKS. This vanilla version has very minimal changes to the upstream Kubeflow manifests. Here are the changes
```
Comment (surajkota) [inline docs/deployment/vanilla/README.md:3]: how to deploy Vanilla Kubeflow on EKS?

same for the heading of this readme Deploying Vanilla Kubeflow on EKS?

wdyt

Comment (akartsky) [reply docs/deployment/vanilla/README.md]: changed to "how to deploy Vanilla Kubeflow on EKS"
but kept the heading

**Review by surajkota — APPROVED** [review]

**Review by mbaijal — APPROVED** [review]

**Pull request merged**